### ISSUE TITLE
Type inference for strings containing numbers

### ISSUE THREAD

**Issue by jakevdp** [vega/altair#296]
An issue came up here: https://github.com/vega/vega-lite/issues/1916

Essentially, the user had passed-in strings containing digits, and was confused that Altair inferred nominal rather than quantitative data.

Should we do something differently here? I see two options:

- for string inputs, detect if all the inputs can be interpreted as numbers, and use quantitative automatically.
- for string inputs, detect if alll the inputs can be interpreted as numbers, and if so warn the user that nominal is always used for strings.

Thoughts?

**Comment by curran** [comment]
Related thread on D3 https://github.com/d3/d3-dsv/issues/6

**Comment by ellisonbg** [comment]
I am fine with us trying to interpret strings as numbers and defaulting back to nominal.

**Comment by yashagrawal3** [comment]
I agree with @ellisonbg , It would be nice to interpret strings as number with also providing an option to convert back to nominal type. Most of discussions in abovementioned links also suggest this.
 Do you think it's still relevant, I am willing to work on this. @jakevdp @ellisonbg

**Comment by dangotbanned** [comment]
Closing as stale
